easybuild/easyconfigs/l/libavif/libavif-1.1.1-GCCcore-13.2.0.eb

```diff
@@ -28,6 +28,10 @@ builddependencies = [
     ('libyuv', '20241125'),
```

Comment:
```suggestion
```

Also needs to be removed from build dependencies

Reply:
My bad